terraform/shared/shared.tf.json.template.py

```diff
@@ -175,21 +189,36 @@
             }
         },
         'aws_cloudtrail': {
+            # FIXME: Disable original CloudTrail trail #4832
+            #        https://github.com/databiosphere/azul/issues/4832
             'shared': {
                 **provider_fragment(config.cloudtrail_trail_region),
                 'name': 'azul-shared',
                 's3_bucket_name': '${aws_s3_bucket.shared_cloudtrail.id}',
                 'enable_log_file_validation': True,
                 'is_multi_region_trail': True,
-                'cloud_watch_logs_group_arn': '${aws_cloudwatch_log_group.cloudtrail.arn}:*',
+                'enable_logging': False,
+            },
+            'trail': {
+                'name': config.qualified_resource_name('trail'),
+                's3_bucket_name': '${aws_s3_bucket.trail.id}',
+                'enable_log_file_validation': True,
+                'is_multi_region_trail': True,
+                'cloud_watch_logs_group_arn': '${aws_cloudwatch_log_group.trail.arn}:*',
                 'cloud_watch_logs_role_arn': '${aws_iam_role.cloudtrail.arn}'
```

Comment:
```suggestion
                'cloud_watch_logs_role_arn': '${aws_iam_role.trail.arn}'
```



Ideally, all CloudTrail-related resources should have the `trail` TF resource name and a fully-qualified real-world name like `azul-trail-dev`. Except the trail bucket, that is.